### ISSUE TITLE
Is it possible to disable this pop-up?

### ISSUE THREAD

**Issue by yoghurt-x86** [fuhsjr00/bug.n#241]
If i open up a fullscreen app, my screen might change resolution and results in this pop-up:
![image](https://user-images.githubusercontent.com/25183723/77328901-802af700-6d1d-11ea-97b1-aff10e9fc547.png)
The pop-up takes focus away from the fullscreen app which makes it exit fullscreen, so i can't use the app. 
Would be nice if bug.n could just ignore it and run as if nothing changed.

**Comment by jfecher** [comment]
I had the same issue and searched through the code and found the message was guarded by the config option:

```
`Config_monitorDisplayChangeMessages=ask`

If set to "ask", bug.n reacts on `WM_DISPLAYCHANGE` system messages asking to 
reset the monitor configuration, e.g. if a monitor is dis-/connected or the 
monitor resolution is changed. If set to "on", bug.n acts without asking, if 
set to "off" or false (`=0`), bug.n ignores these system messages.
```

So set it to either on or off to remove the popup.  If you're trying to fullscreen an app like a game that wants resolution control I'd set it to on

**Comment by schkr** [comment]
Agreed. Set to "on", and let bug.n take care of it. 9+ Monitor config with 4 Tags... set to "on" :)
